homeassistant/components/unmanic/sensor.py

```diff
@@ -0,0 +1,126 @@
+"""Sensor platform for Unmanic."""
+from homeassistant.components.sensor import SensorEntity, SensorEntityDescription
+from homeassistant.helpers.typing import StateType
+
+from .const import DOMAIN, NAME
+from .entity import UnmanicEntity
+
+SENSORS: tuple[SensorEntityDescription, ...] = (
+    SensorEntityDescription(
+        key="total-workers",
+        name=f"{NAME} Total Workers",
+        icon="mdi:account-hard-hat",
+        native_unit_of_measurement="workers",
+        entity_registry_enabled_default=True,
+    ),
+    SensorEntityDescription(
+        key="active-workers",
+        name=f"{NAME} Active Workers",
+        icon="mdi:account-hard-hat",
+        native_unit_of_measurement="workers",
+        entity_registry_enabled_default=True,
+    ),
+    SensorEntityDescription(
+        key="queue",
+        name=f"{NAME} Pending Tasks",
+        icon="mdi:clipboard-text-clock",
+        native_unit_of_measurement="files",
+        entity_registry_enabled_default=True,
+    ),
+    SensorEntityDescription(
+        key="history",
+        name=f"{NAME} Completed Tasks",
+        icon="mdi:clipboard-check",
+        native_unit_of_measurement="files",
+        entity_registry_enabled_default=True,
+    ),
+)
+
+
+async def async_setup_entry(hass, entry, async_add_devices):
+    """Unmanic setup sensor entity."""
+    coordinator = hass.data[DOMAIN][entry.entry_id]
+
+    async_add_devices(
+        [UnmanicSensor(coordinator, entry, description) for description in SENSORS],
+        True,
+    )
+
+
+class UnmanicSensor(UnmanicEntity, SensorEntity):
+    """Unmanic Sensor class."""
+
+    def __init__(self, coordinator, config_entry, description) -> None:
+        """Initialize."""
+        self.coordinator = coordinator
+        self.config_entry = config_entry
```

Comment:
Do you need to set this? This will be set via `super()`, so you should be able to remove this.